yarn-project/txe/src/oracle/txe_oracle.ts

```diff
@@ -257,8 +257,12 @@ export class TXE implements IUtilityExecutionOracle, IPrivateExecutionOracle {
     return values;
   }
 
-  utilityDebugLog(message: string, fields: Fr[]): void {
-    this.logger.verbose(`${applyStringFormatting(message, fields)}`, { module: `${this.logger.module}:debug_log` });
+  utilityDebugLog(levelNumber: number, message: string, fields: Fr[]): void {
+    if (!LogLevels[levelNumber]) {
+      throw new Error(`Invalid debug log level: ${levelNumber}`);
+    }
+    const level = LogLevels[levelNumber];
+    this.logger[level](`${applyStringFormatting(message, fields)}`, { module: `${this.logger.module}:debug_log` });
```

Comment:
```suggestion
  utilityDebugLog(level: number, message: string, fields: Fr[]): void {
    if (!LogLevels[level {
      throw new Error(`Invalid debug log level: ${level}`);
    }
    const levelName = LogLevels[level];
    this.logger[levelName](`${applyStringFormatting(message, fields)}`, { module: `${this.logger.module}:debug_log` });
```

Everywhere else the level is the number, we should not reverse the convetion here